SB2018/2018-02-07.md

```diff
@@ -0,0 +1,37 @@
+---
+to: All Teams
+subject: January Kit Update
+---
+
+Hello there!
+
+Today's the day we're releasing the first round of kit updates for this year's competition! It's been a long time in the making, and hopefully makes your development life a lot easier!
+
+# Kit updates
+This release contains many improvements to the kit, and what you can do with it. The most notable changes are:
+- Reduce the verbosity of logs
+- New coordinate systems
+- Support for [custom Arduino commands](https://docs.sourcebots.co.uk/api/servo-assembly/arduino-commands/)
+
+A more detailed changelog for this release is available from [the docs](https://docs.sourcebots.co.uk/updates/jan-2018/).
+
+To update your kit, you will need to flash your SD card. Instructions on [how to flash an SD card](https://docs.sourcebots.co.uk/kit/pi/sd-card/), and the related [download links](https://docs.sourcebots.co.uk/updates/jan-2018/), are available from [the docs](https://docs.sourcebots.co.uk/kit/pi/#updating-your-pi). If you have any issues updating, be sure to consult the documentation, or [contact us](techsupport@sourcebots.co.uk).
+
+Future updates may be installed using [an update file](https://docs.sourcebots.co.uk/kit/pi/update-file/).
```

Comment:
maybe clarify further that this cannot be updated using an update file? Or will this be obvious by there not being an update file for them to use?

Reply:
We won't distribute the update file, so I think this is ok as-is.